packages/cross_file/lib/src/web_helpers/web_helpers.dart

```diff
@@ -31,7 +31,7 @@ Element ensureInitialized(String id) {
   if (target == null) {
     final Element targetElement = Element.tag('flt-x-file')..id = id;
 
-    querySelector('body').children.add(targetElement);
+    querySelector('body')!.children.add(targetElement);
```

Comment:
is `querySelector('body')` guaranteed to be non-null?

Reply:
It is, when using flutter's default index.html :)

(We need a spot in the DOM to to add the element, and I don't want to look for it in funky places)

Reply:
Is it possible that a non flutter default index.html is used?
do you think it would be nice to have an assert() with error message to help better debug werid issues where "body" does not exist in the html file?